packages/enzyme-adapter-utils/src/createMountWrapper.jsx

```diff
@@ -110,10 +110,12 @@ export default function createMountWrapper(node, options = {}) {
     props: PropTypes.object.isRequired,
     wrappingComponentProps: PropTypes.object,
     context: PropTypes.object,
+    forwardedRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
```

Comment:
can this perhaps use `ref` from https://www.npmjs.com/package/airbnb-prop-types ?

Reply:
Should work: 60954e385e8f04dc2c13fd0b8dd78f4268d0abe8